packages/expo-dev-launcher/Logs/EXDevLauncherRemoteLogsManager.swift

```diff
@@ -0,0 +1,51 @@
+import Foundation
+
+class EXDevLauncherRemoteLogsManager {
+  private var batch: [[String: Any]] = []
+  private let url: URL
+  
+  init(withUrl url: URL) {
+    self.url = url
+  }
+  
+  func deferError(exception: NSException) {
+    batch.append([
+      "level": "error",
+      "body": [
+        "message": exception.description,
+        "stact": exception.callStackSymbols.joined(separator: "\n")
+      ],
+      "includesStack": true
+    ])
+  }
+  
+  func deferError(message: String) {
+    batch.append([
+      "level": "error",
+      "body": message,
+      "includesStack": false
+    ])
+  }
+  
+  func sendSync() {
+    guard let data = try? JSONSerialization.data(withJSONObject: batch, options: []) else {
+      return
```

Comment:
perhaps we should removeAll here anyway? otherwise this will continue to fail if called again